DUI3-DX/Connectors/Revit/Speckle.Connectors.RevitShared/Bindings/SendBinding.cs

```diff
@@ -65,6 +65,8 @@ public void CancelSend(string modelCardId)
     CancellationManager.CancelOperation(modelCardId);
   }
 
+  public SendBindingUICommands Commands { get; }
```

Comment:
Yeah no keen as we're kind of using this as servicelocator pattern

Reply:
Agreed, I don't like this either much... I'm sure we can come up with something better